app/scripts/controllers/network/network-controller.test.ts

```diff
@@ -384,23 +498,95 @@ class NetworkCommunications {
     );
   }
 
-  #validateMockResponseBody(mockResponseBody) {
-    const invalidProperties = Object.keys(mockResponseBody).filter(
-      (key) =>
-        key !== 'httpStatus' && !JSONRPC_RESPONSE_BODY_PROPERTIES.includes(key),
-    );
-    if (invalidProperties.length > 0) {
-      throw new Error(
-        `Mock response object ${inspect(
-          mockResponseBody,
-        )} has invalid properties: ${inspect(invalidProperties)}.`,
-      );
-    }
+  /**
+   * The number of times to mock `eth_blockNumber` by default. Customized
+   * for Infura.
+   */
+  protected getDefaultNumTimesToMockEthBlockNumber(): number {
+    return 0;
+  }
+}
+
+/**
+ * Handles mocking requests made by NetworkController for an Infura network.
+ */
+class InfuraNetworkCommunications extends NetworkCommunications<InfuraNetworkCommunicationsOptions> {
+  /**
+   * Constructs an InfuraNetworkCommunications.
+   *
+   * @param args - The arguments.
+   * @param args.infuraProjectId - TODO.
+   * @param args.infuraNetwork - TODO.
```

Comment:
what are these TODO comments for? 👀 

@mcmire

Reply:
Oops, good catch, thanks. Fortunately I'm going to update this test file again in another PR very soon so I will address them then.

Reply:
👍